app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -204,7 +204,7 @@ def async_state_done(current_async_state)
       )
       log_idv_verification_submitted_event(
         success: form_response.success?,
-        failure_reason: irs_attempts_api_tracker.parse_failure_reason(form_response),
+        failure_reason: form_response.errors,
```

Comment:
This would produce a different result from `error_details`, wouldn't it? Why do we use `errors` in some places and `to_h[:error_details]` in others?

Reply:
Some of the forms only have errors, and don't have any error details, so if we did error details it would give us empty results

Reply:
None of this data is used at all right now so it's all kind of moot?

I guess another option would be to remove `failure_reason` across the board from all events, and then we could add it back in and format it as needed at some undetermined point in the future?